setup.py

```diff
@@ -33,7 +33,7 @@ class _pycurlFake(object):
 scripts = ['scripts/urlgrabber']
 data_files = [
     ('share/doc/' + name + '-' + version, ['README','LICENSE', 'TODO', 'ChangeLog']),
-    ('libexec', ['scripts/urlgrabber-ext-down']),
+#    ('libexec', ['scripts/urlgrabber-ext-down']),
```

Comment:
Why did you do this?

Reply:
Oops! We actually introduced this change here by mistake. I'm updating the PR just now to revert this.

Thanks for reviewing @Conan-Kudo :+1:

Reply:
This got added again.